### PR TITLE
added introduction blog documentation

### LINKED ISSUE #207: Introduction to DP
## Where?
Part of our documentation 

## Who?
Someone looking into pydp for the first time

## What?
- Introduction of DP from the basic concepts to technical explanation. // might include pictorial explanation 
- Where one can learn ahead
- Application of DP 
- Where is it currently used 

## Additional Context
Borrowing some text from the DP blog series?

### PR DESCRIPTION
**Pull request by shivaylamba**
## Description
Converted the article (https://docs.google.com/document/d/17DseJokINbxJbBTBMJursrnUcvv_pgOeDQDtVTDHGpU/edit) written by @TravellerOnCode  into rst format to be added to the docs. 
Please review @chinmayshah99 

## Affected Dependencies
NA

## How has this been tested?
Tested by running on online rst viewers and editors. 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by chinmayshah99 — DISMISSED** [review]

**Review by midopooler — DISMISSED** [review]
LGTM

**Comment by TravellerOnCode** [comment]
Suggested Changes:
* Centre Align all images
* Update the caption (changes has been made in the original doc)
* Under the section: DIFFERENTIAL - PRIVACY IN REAL WORLD, just add "OpenDp by Harvard University and Microsoft "

Rest looks good, feel free to make any changes in the layout as per your choice.

File: docs/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+* If the dataset is large, the type of queries that can be drawn from the dataset cannot be predicted. This makes any dataset prone to external attacks.
+
+Hence, this process is prone to risk and is considered as fundamentally wrong. Netflix once released a challenge for everyone to build up the best recommendation engine. For this they released an anonymized dataset of 100 million movie ratings from half a million users. So, they did not publicly release any data that could lead to the identification of the users. 
+
+.. figure:: https://user-images.githubusercontent.com/19529592/91381064-14a50280-e844-11ea-9dd0-1af088c3924d.png
+    :alt: netflix
+    :align: center
+    
+    < a click from Secure AI Course>
```
Comment (TravellerOnCode) [inline docs/introduction.rst]: Image Credits: Secure and Private AI (Udacity)

File: docs/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+Local Differential Privacy
+-----
+
+In local differential privacy the random noise is applied at the start of the process(local) level i.e when the data is sent to the data curator/aggregator. If the data is too confidential, generally the data generators do not want to trust the curator and hence add noise to the dataset beforehand. This is adopted when the Data Curator cannot be completely trusted.
+
+.. figure:: https://user-images.githubusercontent.com/19529592/91381482-1e7b3580-e845-11ea-9419-cd6bdbbd9dbf.png
+    :alt: local
+    :align: center
+    
+    (from google images)
```
Comment (TravellerOnCode) [inline docs/introduction.rst]: Image Credit: Google Images

File: docs/introduction.rst [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+DIFFERENTIAL - PRIVACY IN REAL WORLD
+============
+
+Differential Privacy ensures privacy of all sorts of data which can be used by anyone to draw insights which can help them run their business. In the present world, Differentially Private Data Analysis is widely used and these are implemented by using various libraries. 
+
+`PyDP <https://github.com/OpenMined/PyDP>`_ by OpenMined is a Python Wrapper for Differential Privacy which allows all sorts of users to use Differential Privacy in their Projects. Apart from this there are various other real-world cases of Differential Privacy from Medical Imaging to Geolocation search. These have been covered in this `blogpost <https://blog.openmined.org/use-cases-of-differential-privacy>`_  by OpenMined.
+
+SOME OTHER LIBRARIES FOR DP
+
+* `OpenDp  <https://github.com/opendifferentialprivacy>`_ by Harvard University
```
Comment (TravellerOnCode) [inline docs/introduction.rst]: OpenDp by Harvard University and Microsoft

File: docs/introduction.rst [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+This is where “Differential Privacy” comes into the picture, a smarter way to a more secure and private AI. According to Andrew Trask, Founder at OpenMined - “Differential Privacy is the process to answer questions or solve problems using the data that we cannot see.” In this way researchers from all over the world can use private data in their research work without identifying the individual.
+
+.. figure:: https://user-images.githubusercontent.com/19529592/91377299-b58fbf80-e83c-11ea-9b56-a068ea3155c6.png
+    :alt: my-picture1
+    :align: center
```
Comment (TravellerOnCode) [inline docs/introduction.rst:20]: add the caption:
(Privacy Preserving AI by Andrew Trask | MIT Deep Learning Series )

**Review by TravellerOnCode — CHANGES_REQUESTED** [review]
Suggested Changes:

* Update the caption for the Images
* Under the section: DIFFERENTIAL - PRIVACY IN REAL WORLD, just add "OpenDp by Harvard University and Microsoft "

Rest looks good, feel free to make any changes in the layout as per your choice.

**Comment by 8bitmp3** [comment]
Awesome PR @shivaylamba! Are you open to edits to the text? There are a few things that stand out that could be polished.

**Comment by chinmayshah99** [comment]
@8bitmp3 please go ahead with the suggestions! You can review the PR

**Comment by shivaylamba** [comment]
> Awesome PR @shivaylamba! Are you open to edits to the text? There are a few things that stand out that could be polished.

Yes @8bitmp3 Please let me know regarding any changes that I should make!

**Comment by shivaylamba** [comment]
@TravellerOnCode I have made the suggested changes. Also I tried to centrally align the images but it always defaults to aligning to the left. I had a look at this open issue from 7 years back : https://github.com/github/markup/issues/163

**Review by chinmayshah99 — APPROVED** [review]

**Comment by chinmayshah99** [comment]
Fixes #207

**Pull request merged**